WordPress/UITests/JetpackUITests-Info.plist

```diff
@@ -20,7 +20,5 @@
 	<string>????</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
-	<key>NSPrincipalClass</key>
-	<string>${PRODUCT_NAME}.TestObserver</string>
```

Comment:
@jostnes I'd recommend keeping this setup and skipping the `executeWithRetries(disableAutoFillPasswords)` in `testBundleWillStart` from `TestObserver`. 

I'm saying this because it's easier to find the `TestObserver` setup in the code than in this `Info.plist` and hopefully we'll be able to go back to that setup when Apple fixes their Simulator bug (assuming my theory about it being a simulator bug is correct)

Reply:
yup, this is a temporary removal to save testing time and skip the part where the test tries to turn it off and fails every time because of the simulator bug. this will be re-added before merging.